### DISCUSSION TITLE
conf gnss Issue

### DISCUSSION THREAD

**Discussion by Adgbeta** [centerclick/feedback#112] [Q&A]
I don't know if it's a bug, some time ago I disabled GLONASS and activated beidou, now I want to activate GLONASS and turn off beidou. I was able to do it with the CLI command conf gnss, command was executed without issues, but when I go to the GPS web interface still showing beidou satellites lock in but not show GLONASS, and the GPS status area show beidou disabled and GLONASS enable. 

I tried to restart, shutdown for an hour, as soon turn on only get 3d fix using GPS, Galileo and beidou, but not sign of GLONASS, 

GNSS Systems	 show GPS: enabled, Galileo: enabled, GLONASS: enable , BeiDou: disabled.

I am using last software v2.17.0, NTP250 uBlox MAX-M10S

Thanks in advance for your help.

**Comment by dave4445** [comment]
#113
